rps.c

```diff
@@ -0,0 +1,122 @@
+//Anya Lauria
```

Comment:
Your code gave out six warnings during compile time - these can screw up your logic, so be careful. And for future reference, try and take out warnings from your code.